ydb/docs/en/core/devops/manual/initial-deployment.md

```diff
@@ -131,7 +131,7 @@ sudo usermod -aG disk ydb
 
   To streamline the next setup step, it makes sense to use the same disk labels on cluster servers having the same disk configuration.
 
-2. Format the disk by this command built-in the `ydbd` executable:
+2. Clear the disk by this command built-in the `ydbd` executable:
```

Comment:
Let's indeed add a `{% note warning %}` saying that the pre-existing data will be lost?